pt-BR/mail/chrome/messenger/aboutSupportMail.dtd

```diff
@@ -1,26 +1,26 @@
-<!ENTITY aboutSupport.appBasicsAppBuildID "Application Build ID">
+<!ENTITY aboutSupport.appBasicsAppBuildID "ID da versão da aplicação">
 
-<!ENTITY aboutSupport.accountsTitle "Mail and News Accounts">
-<!ENTITY aboutSupport.showPrivateData.mainText "Include account names">
-<!ENTITY aboutSupport.showPrivateData.explanationText "(possibly identifying information)">
+<!ENTITY aboutSupport.accountsTitle "E-mail e novas contas">
+<!ENTITY aboutSupport.showPrivateData.mainText "Incluir nomes das contas">
+<!ENTITY aboutSupport.showPrivateData.explanationText "(provavelmente identificando informações)">
 <!ENTITY aboutSupport.accountsID "ID">
-<!ENTITY aboutSupport.accountsName "Name">
-<!ENTITY aboutSupport.accountsIncomingServer "Incoming server">
-<!ENTITY aboutSupport.accountsOutgoingServers "Outgoing servers">
-<!ENTITY aboutSupport.accountsServerName "Name">
-<!ENTITY aboutSupport.accountsConnSecurity "Connection security">
-<!ENTITY aboutSupport.accountsAuthMethod "Authentication method">
-<!ENTITY aboutSupport.filterCount "Filter Count">
-<!ENTITY aboutSupport.accountsDefault "Default?">
+<!ENTITY aboutSupport.accountsName "Nome">
+<!ENTITY aboutSupport.accountsIncomingServer "Servidor de entrada">
+<!ENTITY aboutSupport.accountsOutgoingServers "Servidores de saída">
+<!ENTITY aboutSupport.accountsServerName "Nome">
+<!ENTITY aboutSupport.accountsConnSecurity "Segurança da conexão">
+<!ENTITY aboutSupport.accountsAuthMethod "Método de autenticação">
+<!ENTITY aboutSupport.filterCount "Qtde de filtros">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY aboutSupport.filterCount "Contagem de filtros">